purchase_request_usage_department/tests/test_purchase_request_usage_department.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestPurchaseRequest(TransactionCase):
+
+    def setUp(self):
+        super(TestPurchaseRequest, self).setUp()
+        self.pr_model = self.env['purchase.request']
+        self.prl_model = self.env['purchase.request.line']
+        self.usr_model = self.env['res.users']
+        self.dep_model = self.env['hr.department']
+        self.empee_model = self.env['hr.employee']
+        self.usage_model = self.env['purchase.product.usage']
+        self.usage = self.usage_model.create({
+            'name': 'Personal',
+            'code': 'Ps',
+        })
+        self.usage_2 = self.usage_model.create({
+            'name': 'Marketing',
+            'code': 'Mk',
+        })
+        dept_dict = {
+            'name': 'department in usage',
+            'usage_id': self.usage.id
+        }
+        self.department_usage = self.dep_model.create(dept_dict)
+        dept_dict = {
+            'name': 'department with no usage',
+            'usage_id': self.usage_2.id
+        }
+        self.department_usage_2 = self.dep_model.create(dept_dict)
+        user_dict = {
+            'name': 'User test',
+            'login': 'u@example.com',
+            'email': 'u@example.com',
+            'groups_id': [(4, self.env.ref(
```

Comment:
Here You can Directly get Id Like `self.ref( 'purchase_request.group_purchase_request_user')`